eva_submission/eload_preparation.py

```diff
@@ -154,14 +159,36 @@ def detect_metadata_attributes(self):
     def find_genome(self):
         scientific_name = self.eload_cfg.query('submission', 'scientific_name')
         analyses = self.eload_cfg.query('submission', 'analyses')
+
+        contig_alias_url, contig_alias_user, contig_alias_pass = get_contig_alias_db_creds_for_profile(
+            cfg['maven']['environment'], cfg['maven']['settings_file'])
+
+        contig_alias_payload = []
+
         if scientific_name:
             for analysis_alias in analyses:
                 assembly_accession = self.eload_cfg.query('submission', 'analyses', analysis_alias, 'assembly_accession')
-                assembly_fasta_path, assembly_report_path = get_reference_fasta_and_report(scientific_name, assembly_accession)
+
+                if NCBIAssembly.is_assembly_accession_format(assembly_accession):
+                    contig_alias_payload.append(assembly_accession)
+
+                assembly_fasta_path, assembly_report_path = get_reference_fasta_and_report(scientific_name,
+                                                                                           assembly_accession)
                 if assembly_report_path:
                     self.eload_cfg.set('submission', 'analyses', analysis_alias, 'assembly_report', value=assembly_report_path)
                 else:
                     self.warning(f'Assembly report was not set for {assembly_accession}')
                 self.eload_cfg.set('submission', 'analyses', analysis_alias, 'assembly_fasta', value=assembly_fasta_path)
+
+            for assembly in contig_alias_payload:
```

Comment:
Since we are interfacing with service requests over the network, I suggest moving the code in this loop to a separate function that is annotated with retry (see elsewhere for examples)...